src/openapi.ts

```diff
@@ -147,6 +147,17 @@ export function OpenAPIRouter(options?: RouterOptions): OpenAPIRouterSchema {
         })
       })
     }
+
+    if (options?.aiPlugin !== null) {
+      router.get('/.well-known/ai-plugin.json', () => {
+        return new Response(JSON.stringify(options?.aiPlugin), {
```

Comment:
I have mixed feelings on this, because it sort of encourages that you don't use auth and such. I have not explore more how auth would look in a convenient manner.

Reply:
I agree that is it nice to have this though. I suppose since it is default that is probably known and you can always specify the values. I will make this change!

Reply:
I wouldn't mind guidance or help on documenting this either heh